metrics-core/src/main/java/software/amazon/swage/metrics/record/cloudwatch/CloudWatchRecorder.java

```diff
@@ -79,7 +71,9 @@
 
     private static final Logger log = LogManager.getLogger(CloudWatchRecorder.class);
 
-    // Count of metric datum to batch together in one CloudWatch call
+    // Count of metric data to batch together in one CloudWatch call
```

Comment:
data vs datum

Which is right?  It is impossible to knows.

Reply:
Count of things to batch together...

'data' is right